[OSPO Book] Create glossary.md (#286)
todogroup/ospology: ospo-book/chapters/glossary.md

## Glossary

Welcome to the glossary for the Open Source Program Offices (OSPOs) Book. As you may know, the world of open source is vast and complex, 
with many different terms and concepts that can be difficult to navigate. Our aim with this glossary is not to provide a definitive or general definition 
of these terms, but to clarify what we mean specifically under the context of this book.

We believe that it is important to be concise and clear in our definitions, and to reference existing descriptions and resources whenever possible. 
This will help us avoid creating a glossary that is overly complex or difficult to understand, and will ensure that our readers have access to the best 
information and resources available.

> 

Author: ```suggestion
**Free Software:** Free Software refers to freedom, not price. It guarantees its users the essential four freedoms (Use, Study, Share, Improve) The absence of at least one of these freedoms means an application is proprietary, so non‐Free Software. Learn more: https://fsfe.org/freesoftware/freesoftware.en.html
```

* **InnerSource:** is the use of open source software development best practices and the establishment of an open source-like culture within organizations
for the development of its non-open-source and/or proprietary software. https://en.wikipedia.org/wiki/Inner_source

> * **InnerSource Principles:** InnerSource Principles are a set of guidelines that provide a framework InnerSource Principles provide a framework for organizations 

Reviewer: I think some rewording is needed: "InnerSource Principles are a set of guidelines that provide a framework for organizations to tap into the collective knowledge and expertise of their employees, and to create a culture of collaboration and innovation."

Author: rewording committed! Thanks @supriyachitale

to tap into the collective knowledge and expertise of their employees, and to create a culture of collaboration and innovation. 
Learn more about InnerSource principles: https://innersourcecommons.org/learn/

* **Knowledge Sharing:** Knowledge sharing refers to the exchange of information and expertise between individuals, teams, and organizations. 
Knowledge sharing is often a key component of open source and innerSource development practices. Learn more about knowledge sharing:https://en.wikipedia.org/wiki/Knowledge_sharing

* **Open Source:** Open source refers to a type of software that allows users to freely use, modify, and distribute 
the source code of a program that must comply with certain distribution criteria. Learn more about open source: https://opensource.org/osd/

> * **Open Source Culture:** Open source culture refers to a set of values and practices that promote collaboration, transparency, 

Author: From contributor discussion: not mix innersource with open source culture. Maybe changing the existing link to this that mentions the "4 opens" https://openinfra.dev/four-opens/

and community-driven development. It is often associated with the open source software movement, but can be applied to other 

> areas as well, such as open science, open hardware, and open education. Learn more about open source culture: 

Author: ```suggestion
areas as well, such as open science, open hardware, and open education. For this book, we will base on the *four opens philosophy*, created by the OpenStack community. Learn more about: https://openinfra.dev/four-opens/
```

> https://en.wikipedia.org/wiki/Inner_source

Reviewer 2: Is this the right URL for Open source culture?

> https://en.wikipedia.org/wiki/Inner_source

Author: ```suggestion

```

* **Open Source Project Dynamics:** Open source project dynamics refers to the complex social and technical interactions that occur within 
open source communities. This includes things like peer review, community-driven governance, and decentralized 
decision making. Understanding these dynamics is important for anyone looking to participate in or manage open source projects. 

* **Open Works:** Open Works refers to the meaning of “Open” with respect to work products and projects that are released into the public domain 
or under a license recognized by an entity such as, but not limited to, the Open Source Initiative (OSI), the Free Software Foundation GNU, 
the Creative Commons (CC) or the Open Knowledge Foundation (OKF) with the intent of promoting a robust community of collaboration around 
the work itself and other work that it may interoperate with. Learn more about Open Works: http://openworkdefinition.com/

> 

Reviewer 3: I would also included the OSPO term in the glossary. We could reuse the definition from chapter 1.

Author: Good idea! FYI: There is a ospodefinition.org repo in TODO that host the OSPO definition (a PR to update the description has been submitted, and hopefully will mirror the definition from the OSPO book) https://github.com/todogroup/ospodefinition.org

> 

Reviewer 3: Futhermore I think it would be good to also include the term Free software in the glossary. There is sometimes some confusion about the difference between free software and open source software.

## Glossary

Welcome to the glossary for the Open Source Program Offices (OSPOs) Book. As you may know, the world of open source is vast and complex, 
with many different terms and concepts that can be difficult to navigate. Our aim with this glossary is not to provide a definitive or general definition 
of these terms, but to clarify what we mean specifically under the context of this book.

We believe that it is important to be concise and clear in our definitions, and to reference existing descriptions and resources whenever possible. 
This will help us avoid creating a glossary that is overly complex or difficult to understand, and will ensure that our readers have access to the best 
information and resources available.

> **Free Software:** Free Software refers to freedom, not price. It guarantees its users the essential four freedoms (Use, Study, Share, Improve) The absence of at least one of these freedoms means an application is proprietary, so non‐Free Software. Learn more: https://fsfe.org/freesoftware/freesoftware.en.html

Author: ```suggestion
* **Free Software:** Free Software refers to freedom, not price. It guarantees its users the essential four freedoms (Use, Study, Share, Improve) The absence of at least one of these freedoms means an application is proprietary, so non‐Free Software. Learn more: https://fsfe.org/freesoftware/freesoftware.en.html
```
